doc/development/style_guide.md

```diff
@@ -577,6 +577,43 @@ gs.run_command("r.slope.aspect", elevation=input_raster, slope=slope, env=env)
 This approach makes the computational region completely safe for parallel
 processes as no region-related files are modified.
 
+#### Changing raster mask
+
+The MaskManager in Python API is provides a way for tools to change, or possibly
+to ignore, a raster mask for part of the computation.
+
+Without parameters, MaskManager modifies the global system environment for the
+tool
```

Comment:
Maybe add the line from documentation "The mask is deactivated at the end of the context by the context manager" or add a comment to the example.

Reply:
Maybe also clarify it ignores the current mask.